cypress/e2e/listingDetails/completedLotteryListing.e2e.ts

```diff
@@ -0,0 +1,96 @@
+const listings = {
+  COMPLETED_LOTTERY: {
+    id: "a0W8H000000AmpKUAS",
+  },
+}
+
+const MOBILE_VIEWPORT_HEIGHT = 680
+const MOBILE_VIEWPORT_WIDTH = 420
+
+const visitListing = (mobile, language) => {
+  const langPart = language ? `/${language}` : ""
+  if (mobile) {
+    cy.viewport(MOBILE_VIEWPORT_WIDTH, MOBILE_VIEWPORT_HEIGHT)
+  }
+  cy.visit(`${langPart}/listings/${listings.COMPLETED_LOTTERY.id}?react=true`)
+}
+
+const clickLotteryResultsButton = (mobile) => {
+  if (mobile) {
+    cy.get(".detail-header__title").contains("Lottery").click()
```

Comment:
similar comment here as line 26. let's not use classes or id's to get to elements (see guidance [here](https://testing-library.com/docs/react-testing-library/intro/#this-solution))